receiver/awscontainerinsightreceiver/internal/stores/utils.go

```diff
@@ -143,12 +143,16 @@ func TagMetricSource(metric CIMetric) {
 	}
 }
 
-func AddKubernetesInfo(metric CIMetric, kubernetesBlob map[string]interface{}) {
-	needMoveToKubernetes := map[string]string{ci.ContainerNamekey: "container_name", ci.K8sPodNameKey: "pod_name",
-		ci.PodIDKey: "pod_id"}
-
+func AddKubernetesInfo(metric CIMetric, kubernetesBlob map[string]interface{}, retainContainerNameTag bool) {
```

Comment:
I'm assuming add and retain mean the same thing in this context can we use the same naming?

Reply:
Yea I was debating the same. From the end user's perspective, its adding ContainerName as a label so add makes sense.
But from `AddKubernetesInfo`'s perspective, the ContainerName tag was already added previously in the code, so it now just needs to decide is it should retain it or not.

If we think its confusing, Im ok using the same var name everywhere.

Reply:
I'm indifferent, but maybe `include` could be a more pointed verb that clears up the confusion? The issue is that we are dropping the container name tag, right? Retain sounds like the most appropriate verb. Idk, like I said I'm indifferent.